app/views/admin/base/show.html.erb

```diff
@@ -79,14 +84,25 @@
           <%= order.total_price %>
       </div>
 
-      <div class="col-xs-2">
+      <div class="col-xs-2 order-status">
         <%= order.status %>
       </div>
 
-      <div class="col-xs-2">
+      <div class="col-xs-2 order-view">
           <%= link_to "View", order_path(order) %>
       </div>
 
+      <div class="col-xs-2 order-update">
+        <% if order.status == 'ordered' %>
```

Comment:
I think we may be able to use enum methods provided to make this a bit cleaner. Should be able to call status.ordered? .paid? etc. and it returns t/f